js/apps/admin-ui/src/user-federation/shared/Header.tsx

```diff
@@ -99,7 +99,7 @@ export const Header = ({
                 if (!value) {
                   toggleDisableDialog();
                 } else {
-                  field.onChange(value.toString());
+                  field.onChange(`${value}`);
```

Comment:
Why can't we use `toString()` here?

Reply:
we can, this is just shorter and `undefined / null ` proof